samples/cpp/benchmark_app/utils.hpp

```diff
@@ -149,3 +154,122 @@ void convert_io_names_in_map(
     }
     map = new_map;
 }
+
+void verifyBinaryFile(std::ifstream& binaryFile, const std::string& fileName, const unsigned long inputSize);
+template <typename T>
+void verifyNumpyFile(std::ifstream& binaryFile,
+                     const std::string& fileName,
+                     const ov::Shape& inputShape,
+                     const unsigned long inputSize) {
+    auto fullFileSize = static_cast<std::size_t>(binaryFile.tellg());
+    binaryFile.seekg(0, std::ios_base::beg);
+    OPENVINO_ASSERT(binaryFile.good(), "Can not read ", fileName);
+
+    std::string magic_string(6, ' ');
+    binaryFile.read(&magic_string[0], magic_string.size());
+    OPENVINO_ASSERT(magic_string == "\x93NUMPY",
```

Comment:
```suggestion
    std::array<char, 6> ref{'\x93','N','U','M','P','Y'};
    std::array<char, ref.size()> magic_string;
    binaryFile.read(magic_string.data(), magic_string.size());
    OPENVINO_ASSERT(magic_string == ref,
```
Normally std::string::data() returns const char*. Your hack is like applying const cast to that data ptr. I don't think it's a good way

Reply:
Oh, you use the same trick for `header`. So never mind